author	David Herrmann <dh.herrmann@googlemail.com>
commit	f7540d9902bb9006301adfe84c331205606eeb08
	Fri, 21 Sep 2012 13:50:02 +0000 (16:50 +0300)

wiimote: add Wii Balance Board detection

Add the name of the Wii Balance Board so it can be paired with BlueZ like
any other Wii Remote. Reported by Florian Echtler.

diff -cr bluez-4.99.pan.orig/plugins/wiimote.c bluez-4.99.pan/plugins/wiimote.c
*** bluez-4.99.pan.orig/plugins/wiimote.c	Fri Oct 12 08:11:09 2012
--- bluez-4.99.pan/plugins/wiimote.c	Wed Oct 17 06:14:21 2012
***************
*** 75,80 ****
--- 75,81 ----
  	name[sizeof(name) - 1] = 0;
  
  	if (g_str_equal(name, "Nintendo RVL-CNT-01") ||
+ 				g_str_equal(name, "Nintendo RVL-WBC-01") ||
  				(vendor == 0x057e && product == 0x0306)) {
  		DBG("Forcing fixed pin on detected wiimote %s", addr);
  		memcpy(pinbuf, &sba, 6);
